copyspecial.py

```diff
@@ -6,36 +6,86 @@
 # Google's Python Class
 # http://code.google.com/edu/languages/google-python-class/
 
+
+"""
+Assignment: CopySpecial
+
+Description: program that takes in a directory to find special files.
+Shows the absolute path to the files and then prints them to the stdout.
+Two options are allowed. --tozip and --todir.
+--todir copies special files to directory specified
+--tozip runs command to zip files and zips them there
+
+Author: Aaron Jackson
+Github: TimeApollo
+"""
+__author__ = 'TimeApollo'
+
 import sys
 import re
 import os
 import shutil
 import commands
```

Comment:
FYI: the commands module was deprecated in Python 3. It's become the subprocess module.

Reply:
Thanks for the info.